app/views/staff/shared/transactions/_table.html.haml

```diff
@@ -17,9 +17,9 @@
         %th.govuk-table__header
           =t("activerecord.attributes.transaction.disbursement_channel")
         %th.govuk-table__header
-          =t("activerecord.attributes.transaction.provider_id")
+          = "Provider"
```

Comment:
This project uses locales a great deal of the time - how do you come to a decision about when to use the locale and when not to as here?

Reply:
Good point. I try to think of using locale by default whereever possible. When I came to this I couldn't figure out where a table header should be organised in the locale file and forgot to make a note to come back to it! I will have another crack 👍